inventory/environ.py

```diff
@@ -191,6 +198,16 @@ def overrideEnvironmentVars(vars_scope):
     vars_scope["splunk"]["http_enableSSL_cert"] = os.environ.get('SPLUNK_HTTP_ENABLESSL_CERT', vars_scope["splunk"]["http_enableSSL_cert"])
     vars_scope["splunk"]["http_enableSSL_privKey"] = os.environ.get('SPLUNK_HTTP_ENABLESSL_PRIVKEY', vars_scope["splunk"]["http_enableSSL_privKey"])
     vars_scope["splunk"]["http_enableSSL_privKey_password"] = os.environ.get('SPLUNK_HTTP_ENABLESSL_PRIVKEY_PASSWORD', vars_scope["splunk"]["http_enableSSL_privKey_password"])
+    #Used for multisite
+    if 'SPLUNK_SITE' in os.environ or 'site' in vars_scope["splunk"]:
+        vars_scope["splunk"]["site"] = os.environ.get('SPLUNK_SITE', vars_scope["splunk"].get("site"))
+        vars_scope["splunk"]["all_sites"] = os.environ.get('SPLUNK_ALL_SITES', vars_scope["splunk"].get("all_sites"))
+        vars_scope["splunk"]["multisite_master"] = os.environ.get('SPLUNK_MULTISITE_MASTER', vars_scope["splunk"].get("multisite_master"))
+        vars_scope["splunk"]["multisite_master_port"] = os.environ.get('SPLUNK_MULTISITE_MASTER_PORT', vars_scope["splunk"].get("multisite_master_port", 8089))
```

Comment:
Wondering if we need this new env var - in general, we always assume `splunk.svc_port` for management port.

Reply:
Actually, I think it would be helpful to expose a way to connect to remote Splunk instances on non-traditional ports, but I think we should talk about how that would be done? I like the idea of supporting `SPLUNK_MULTISITE_MASTER=cm1` or `SPLUNK_MULTISITE_MASTER=cm1:8089` versus adding yet another environment variable :P 

This could also extend to connecting ephemeral containerized search heads to long-running indexers somewhere else (like EC2) via something like `SPLUNK_INDEXER_URL=idx1:8089,idx2:8090,idx3:18089`

Reply:
I'm fine changing it back to svc_port.  I was thinking about non-traditional ports, but its probably better that until we get a request for it, we shouldn't add another variable.